content/blog/next-level-iac-no-more-clickops/index.md

```diff
@@ -0,0 +1,341 @@
+---
+title: "Next-level IaC: ClickOps No More"
+
+date: 2024-07-15
+draft: false
+social_media: "Many organizations start out using Click-Ops, but find themselves digging a hole they can't get back out of. Pulumi to the rescue!"
+meta_desc: "How to use Pulumi to dig yourself out of a Click-Ops hole."
+meta_image: meta.png
+authors:
+    - troy-howard
+tags:
+    - next-level-iac
+    - python
+    - clickops
+    - boto
+    - import
+---
+
+Many organizations start out using Click-Ops, but find themselves digging a hole they can't find their way out of. Today we're going to show you how to use Pulumi to find your way out.
+
+<!--more-->
+
+Imagine this; you, a valiant keyboard warrior, skilled in the dark arts of Python, just joined a new company to do DevOps... and inherited some existing cloud infrastructure. The person before you just clicked around in AWS making things, instead of using infrastructure-as-code to manage it. Ugh... We call that "Click-Ops". Congratulations, you’ve found yourself in a Click-Ops hole.
```

Comment:
I am not sure semicolon is the correct punctuation mark after "Imagine this"

Reply:
You're right! It should be a colon.